rosbag2_transport/include/rosbag2_transport/record_options.hpp

```diff
@@ -30,6 +30,8 @@ struct RecordOptions
   std::string rmw_serialization_format;
   std::chrono::milliseconds topic_polling_interval;
   std::string node_prefix = "";
+  std::string compression_mode = "NONE";
```

Comment:
Defined constant instead of magic string?

Reply:
This is a direct mapping of the raw options provided from the ros2bag CLI being passed down to the C++ layer.
The ros2bag CLI already ensures that the user can only specify a certain type (and has a default) so maybe we can just make this an empty string.

Reply:
Or give it no default value at all, if it's always passed through?